diff --git a/dev/build/swift-t-settings.sh.template b/dev/build/swift-t-settings.sh.template
index 076964f6c..a78f55dbc 100644
--- a/dev/build/swift-t-settings.sh.template
+++ b/dev/build/swift-t-settings.sh.template
@@ -18,7 +18,7 @@
 # Please fill in all of these variables
 
 # Install root: change to your install location
-export SWIFT_T_PREFIX=/tmp/swift-t-install
+export SWIFT_T_PREFIX=$PREFIX/swift-t
 
 #+====================================+
 #|| Optional configuration variables ||
@@ -79,10 +79,10 @@ SWIFT_T_TRACE_BUILD=0
 # NOTE: Python detection may require PYTHONPATH and PYTHONHOME
 # in the environment
 # Enable Python integration
-ENABLE_PYTHON=0
+ENABLE_PYTHON=1
 # Specify the Python interpreter executable below;
 # if empty, looks for python in PATH
-PYTHON_EXE=""
+PYTHON_EXE="$PYTHON"
 # PYTHON_EXE=$HOME/sfw/Python-2.7.10/bin/python
 
 # Enable R integration
