diff --git a/vistrails/run.py b/vistrails/run.py
index a5034de..aa87590 100644
--- vistrails/run.py
+++ vistrails/run.py
@@ -34,6 +34,16 @@
 ##
 ###############################################################################
 """Main file for the VisTrails distribution."""
+# imports to make sure the new versions of all the QT apis are used
+# this is needed to
+import sip
+sip.setapi('QDate', 2)
+sip.setapi('QDateTime', 2)
+sip.setapi('QString', 2)
+sip.setapi('QTextStream', 2)
+sip.setapi('QTime', 2)
+sip.setapi('QUrl', 2)
+sip.setapi('QVariant', 2)

 import os
 import sys
