--- wscript     2011-04-18 03:42:29.000000000 -0400
+++ waf-wscript-1.8.16  2015-11-10 15:24:28.418439303 -0500
@@ -13,22 +13,23 @@ cairo_version_required = '1.10.0'

 def options(ctx):
   print('  %s/options()' %d)
-  ctx.tool_options('gnu_dirs')
-  ctx.tool_options('compiler_cc')
-  ctx.tool_options('python') # options for disabling pyc or pyo compilation
+  ctx.load('gnu_dirs')
+  ctx.load('gcc')
+  ctx.load('python') # options for disabling pyc or pyo compilation


 def configure(ctx):
   print('  %s/configure()' %d)

   env = ctx.env
-  ctx.check_tool('gnu_dirs')
-  ctx.check_tool('compiler_cc')
-  ctx.check_tool('python')
+  ctx.load('gnu_dirs')
+  ctx.load('gcc')
+  ctx.load('python')
   ctx.check_python_version((3,1,0))
   ctx.check_python_headers()
-  ctx.check_cfg(package='cairo', atleast_version=cairo_version_required,
-                args='--cflags --libs')
+  # ctx.check_cfg(package='cairo', atleast_version=cairo_version_required,
+  #               args='--cflags --libs')
+  # ctx.check_cfg(package='cairo', args='--cflags --libs')

   # add gcc options
   if env['CC_NAME'] == 'gcc':

